src/github.com/couchbase/sync_gateway/rest/changes_api_test.go

```diff
@@ -519,6 +520,65 @@ func TestDocDeduplication(t *testing.T) {
 
 }
 
+func TestIndexChangesMultipleRevisions(t *testing.T) {
+	it := initIndexTester(true, `function(doc) {channel(doc.channel);}`)
```

Comment:
I ran this test against the master branch and it didn't produce any failures -- just curious if you were able to detect the issue with this test or if it was just to validate correct behavior?